examples/README.txt

```diff
@@ -6,6 +6,8 @@ Examples
 End-to-end examples show how you can use PyFluent. If the PyFluent ``ansys-fluent-core`` package is installed on your machine,
 you can download these examples as Python files or Jupyter notebooks and run them locally.
 
+The full collection of PyFluent examples can be found at `pyfluent-examples <https://github.com/ansys/pyfluent-examples>`_.
```

Comment:
```suggestion
A collection of PyFluent examples can be found at `pyfluent-examples <https://github.com/ansys/pyfluent-examples>`_.
```
Technically not all PyFluent examples will necessarily be available there, we may have examples in other repos, and examples such as https://github.com/ansys/pyfluent/tree/main/examples/00-fluent which are only here, and not there